pkgs/applications/misc/ArchiSteamFarm/web-ui/default.nix

```
@@ -2,18 +2,18 @@
 
 buildNpmPackage rec {
   pname = "asf-ui";
-  version = "f70253c96e76fff5c5537c7be57a57de4e273eb8";
+  version = "78188871dfce90fb04096e9fd0b6ce2411312dae";
```

Wouldn't an unstable version make more sense here?
```suggestion
  version = "0-unstable-2024-05-31";
```

---

That's auto generated via the update script and the endpoint we are using doesn't contain that information https://github.com/NixOS/nixpkgs/blob/e64cfbbf9957eba0711a02528311f2a1d75fd384/pkgs/applications/misc/ArchiSteamFarm/web-ui/update.sh#L6 and I am a bit to lazy right now to do an additional API call to get the commit with the date and convert that. We can just download the latest version, because they don't need to be compatible per say. We could download the submodule instead but that breaks more easy.